hello_world/hello_world.cpp

```diff
@@ -2,7 +2,8 @@
 
 using namespace std;
 
-int main() {
-  cout << "Hello world!" << endl;
+int main() 
+{
+  cout << "Hello world!" << endl;//cout statement is used for printing a given output to the screen
```

Comment:
is this required?

Reply:
No its not a must. But comments and intendation will definitely play an important role in helping users to read and analyze your code. That's it is highly recommended.